packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js

```diff
@@ -219,10 +222,24 @@ export default class Inspector extends PureComponent {
         this.setState({toggledPanels: newState});
     };
 
+    getAmountOfValidationErrors = (tab, validationErrors) => {
+        let errors = 0;
```

Comment:
Btw, this could also be a `reduce` (no need to change anything now, just saying)